src/handlers/tests/scoring-rubric-built-ins.test.ts

```diff
@@ -0,0 +1,35 @@
+import Decimal from "decimal.js";
+
+describe("Scoring Rubric Built-ins", () => {
+  let wordScoreCommentDetails: { [key: string]: Decimal };
+
+  beforeEach(() => {
+    wordScoreCommentDetails = {};
+  });
+
+  it("should increment counter for non-built-in words", () => {
+    const words = ["hasOwnProperty", "valueOf", "was", "async", "but", "the", "constructor", "isn", "t", "I", "was"];
+    const ZERO = new Decimal(0);
+
+    const builtIns = new Set(
+      Object.getOwnPropertyNames(Object.prototype).filter((name) => typeof Object.prototype[name] === "function")
+    );
+
+    for (const word of words) {
+      if (!builtIns.has(word)) {
+        const counter = wordScoreCommentDetails[word] || ZERO;
+        wordScoreCommentDetails[word] = counter;
+      }
+    }
+
+    expect(wordScoreCommentDetails).toEqual({
+      was: new Decimal(0),
```

Comment:
This doesn't make sense to me. Was is not a built in property so why is it's count 0 if it's in your test array?

Reply:
because the in the test I've simplified the logic down to the core issue and removed all of the actual scoring, so it's set as zero by default. All of the built-ins are removed from the array of words

```ts
const counter = wordScoreCommentDetails[word] || ZERO;
```

Reply:
I don't understand but based on the non test code changes, and moleculas recommendation I can merge this pull